designs/virtualization/4.5/Uninstall-cnv/uninstall-cnv.md

```diff
@@ -0,0 +1,27 @@
+---
+parent: Virtualization
+version: 4.5
+---
+
+# Uninstall CNV
+
+Ensure that the user is aware that uninstalling this operator will require to delete all VMs as well. That deletion cannot be reversed and data will be lost.
+
+![Operator details page](img/uninstall-0-0.png)
```

Comment:
```suggestion
![Operator details page](img/Uninstall-0-0.png)
```

It looks like all of the images are broken - capitalizing the `U` should fix them.